src/adapters/sqlite.rs

```diff
@@ -46,7 +46,7 @@ fn format_blob(b: ValueRef) -> String {
         Null => "NULL".to_owned(),
         Integer(i) => format!("{}", i),
         Real(i) => format!("{}", i),
-        Text(i) => format!("'{}'", i.replace("'", "''")),
+        i @ Text(_) => format!("'{}'", i.as_str().unwrap().replace("'", "''")),
```

Comment:
@phiresky Because we `match` on `Text(_)`, `i.as_str()` is pretty much guaranteed to succeed. If you want me to add some error handling - let me know.

Reply:
Since sqlite db aren't necessarily utf8 and we don't really care if there is information loss here, i think it would be better to do `Text(i) => format!("'{}'", String::from_utf8_lossy(i)...`